planning_timeout_attempts: 10